Wikipedia/Code/WMFSettingsTableViewCell.m

```diff
@@ -21,6 +27,11 @@ - (void)setTitle:(NSString *)title {
     self.titleLabel.text = title;
 }
 
+- (void)setButtonTitle:(NSString *)buttonTitle {
```

Comment:
not necessary, but you could also override the getter to read the title from the button (and eliminate the need for storage all together).

Reply:
Oh, is the problem you need to set the string before the view is ever on screen?

Reply:
that'll work, thanks for the suggestion 👍